RunLog/RunLog/Sources/Presentation/DetailLog/View/DetailLogView.swift

```diff
@@ -31,7 +31,7 @@ final class DetailLogView: UIView {
     }
     
     /// 지도 위에 오버레이될 무빙트랙 버튼
-    private let movingTrackButton = UIButton(type: .system).then {
+    let movingTrackButton = UIButton(type: .system).then {
         $0.configuration = nil
         
         let attributedTitle = NSAttributedString.RLAttributedString(text: "무빙트랙", font: .Label2, color: .Gray000)
```

Comment:
```suggestion
        let attributedTitle = NSAttributedString.RLAttributedString(text: "무빙트랙", font: .Label2)
```
기본값이 Gray000이라 따로 지정안해 주셔도 됩니다!